src/Chapter20/Listing20.03.AsynchronousHighLatencyUsingTaskAsynchronyPattern.cs

```diff
@@ -41,7 +43,7 @@ public static async Task Main(string[] args)
             Task<int> taskSearch = CountOccurrencesAsync(
              downloadData, findText);
 
-            Console.Write("\nSearching...");
+            Console.Write($"{Environment.NewLine}Searching...");
```

Comment:
```suggestion
            Console.WriteLine("Searching...");
```
Use WriteLine over Write

Reply:
Same here, we can't use `WriteLine` because it wants to append periods to the current line.

Reply:
Thanks for catching!